### PR TITLE
[BACKLOG-38691] - Create and Add Tree Map Visualization to the available list of Chart options in Analyzer

### PR DESCRIPTION
**Pull request by phalguni-a**
RequireJs configuration from #https://github.com/pentaho/pentaho-platform-plugin-common-ui/pull/1709

### CODE REVIEW

File: core/src/main/javascript/pt/webdetails/cgg/resources/define-cfg.js [hunk 1]
```diff
@@ -764,6 +772,7 @@
         "pentaho/visual/models/types/FontStyle",
         "pentaho/visual/models/types/Sides",
         "pentaho/visual/models/types/LabelsOption",
+        "pentaho/visual/models/types/LayoutMode",
```
Comment (dcleao) [inline core/src/main/javascript/pt/webdetails/cgg/resources/define-cfg.js:775]: Did not update changed name.

**Review by dcleao — CHANGES_REQUESTED** [review]

**Comment by buildguy** [comment]
### :warning: Build finished in 3m 22s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 1 test(s) failed:

<details><summary><b>pt.webdetails.cgg.CggGoldSampleTest.testExecuteReports</b> (click to expand)</summary><p>

```
Failed at /build/wingman/workspace/maintenance/wingman/builds/webdetails.cgg.PR-170/core/target/test-classes/test-gold/cgg/sample1/cccRaw-svg-test.js
```

</p></details>

  
Tests run: **2**, Failures: **1**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/cgg-plugin/job/PR-170/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by dcleao — APPROVED** [review]

**Comment by buildguy** [comment]
### :warning: Build finished in 3m 15s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 1 test(s) failed:

<details><summary><b>pt.webdetails.cgg.CggGoldSampleTest.testExecuteReports</b> (click to expand)</summary><p>

```
Failed at /build/wingman/workspace/maintenance/wingman/builds/webdetails.cgg.PR-170/core/target/test-classes/test-gold/cgg/sample1/cccRaw-svg-test.js
```

</p></details>

  
Tests run: **2**, Failures: **1**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/cgg-plugin/job/PR-170/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 12m 20s
Build command:
```console
mvn clean verify -B -e -Daudit -amd
```
---
:ok_hand: All tests passed!
 
Tests run: **2**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/cgg-plugin/job/PR-170/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Pull request closed without merging**